keybuffer.h

```diff
@@ -22,9 +22,9 @@ typedef struct
 typedef struct
 {
     st_key_action_t * const data;        // array of keycodes
-    const int               size;        // buffer size (TODO: rename to capacity)
-    int                     context_len; // number of current keys in buffer (TODO: rename to size)
-    int                     cur_pos;     // current head for circular access
+    const int               capacity;        // buffer size (TODO: rename to capacity)
+    int                     size; // number of current keys in buffer (TODO: rename to size)
+    int                     current_head;     // current head for circular access
```

Comment:
There is no other `head` than the current, so why not just `head`? 
Also rename comments should be removed.

Reply:
Agreed on both points